service/docs/source/geopm_agent_frequency_map.7.rst

```diff
@@ -59,6 +60,15 @@ Policy Parameters
       out of range value or specifying NAN will result in
       a runtime error.
 
+  ``FREQ_GPU_DEFAULT``\ :
+      The operating frequency in units of *Hz* set by the
+      agent on GPUs executing the application.
+      The GPUs not associated with the application
+      will run at the default frequency.  This policy
```

Comment:
What do you mean by "default frequency" here? Do you mean this same policy parameter? If so, the documentation above mentions the default GPU frequency applies only to GPUs running the application. Either way, some clarification is needed above and/or here.